runtime/environment-cpu.yml

```diff
@@ -17,5 +17,11 @@ dependencies:
   - scipy=1.9.1
   - tensorflow-cpu=2.10.0
   - tqdm=4.64.1
+  - imbalanced-learn==0.10.0
+  - matplotlib==3.6.2
+  - opacus==1.2.0
+  - sdmetrics==0.8.1
+  - seaborn==0.12.1
```

Comment:
@ddathan Are you generating plots with matplotlib and seaborn as part of your training code? There will not be a way to get any plots out—only text logs.

Reply:
Hi @jayqi, the plots are needed for our development and final report. Of course we will switch this off for the submission code execution and do not expect to create any plots then

Reply:
In order to make the runtime environment as simple as possible to support solutions, it doesn't make sense for us to include dependencies for detailed reporting. I encourage use to run these reporting tools separately on the outputs of your local evaluation. 

Can you clarify if the `sdmetrics` package is also used in this way?